docs/posts/release-0-2-0.md

```diff
@@ -0,0 +1,255 @@
+# Dioxus v0.2 Release: Router, State Management, and Tooling
+
+> Jan 26, 2022
+
+> [@jkelleyrtp](https://github.com/jkelleyrtp)
+
+> Thanks: [@mrxiaozhuox](https://github.com/mrxiaozhuox) [@autarch](https://github.com/autarch) [@FruitieX](https://github.com/FruitieX) [@t1m0t](https://github.com/t1m0t)
+
+> Thanks: [@t1m0t](https://github.com/t1m0t) for your financial support! 
+
+A few weeks in, and already a ton of awesome changes to Dioxus!
+
+Dioxus is a recently-released library for building interactive user interfaces (GUI) with Rust. It is built around a Virtual DOM, making it portable for the web, desktop, server, mobile, and more. Dioxus looks and feels just like React, so if you know React, then you'll feel right at home.
+
+```rust
+fn app(cx: Scope) -> Element {
+    let (count, set_count) = use_state(&cx, || 0);
+
+    cx.render(rsx! {
+        h1 { "Count: {count}" }
+        button { onclick: move |_| set_count(count + 1), "+" }
+        button { onclick: move |_| set_count(count - 1), "-" }
+    })
+}
+```
+
+# What's new?
+
+A *ton* of stuff happened in this release; 109 commits, 10 contributors, 2 minor releases, and 1 backer on Open Collective (!!!).
+
+The TLDR of the major features:
+
+- We have a new router in the spirit of React-Router [@autarch](https://github.com/autarch)
+- We now have Fermi for global state management in the spirit of [Recoil.JS](https://recoiljs.org)
+- The docs and readme are now translated into Chinese thanks to [@mrxiaozhuox](https://github.com/mrxiaozhuox)
+- Our VSCode Extension and CLI tools now support HTML-to-RSX translation and auto-formatting
+- Dioxus-Web is sped up by 2.5x with JS-based DOM manipulation (3x faster than React)
+
+We also fixed and improved a bunch of stuff - check out the full list down below.
+
+## New Router
+
+We totally revamped the router, switching away from the old yew-router approach to the more familiar [React-Router](http://reactrouter.com). It's less type-safe but provides more flexibility and support for beautiful URLs.
+
+Apps with routers are *really* simple now. It's easy to compose the "Router", a "Route", and "Links" to define how your app is laid out:
+
+```rust
+fn app(cx: Scope) -> Element {
+    cx.render(rsx! {
+        Router {
+            onchange: move |route| log::info!("Route changed to {route}"),
+            ul {
+                Link { to: "/",  li { "Go home!" } }
+                Link { to: "users",  li { "List all users" } }
+                Link { to: "blog", li { "Blog posts" } }
+            }
+            Route { to: "/", "Home" }
+            Route { to: "users",
+                Route { to: "/", "User list" }
```

Comment:
This nested `Route` stuff doesn't work yet, just FYI. You have to spell out the entire route at each level:

```rust
Route { to: "/users", "User list" },
Route { to: "/users/:name", User { } }
```

Also, did you intend to nest everything under the Home route in this example? Because otherwise we have a bunch of routes without a leading slash.